packages/terra-show-hide/tests/jest/ShowHide.test.jsx

```diff
@@ -5,6 +5,8 @@ import ThemeContextProvider from 'terra-theme-context/lib/ThemeContextProvider';
 import { mountWithIntl } from 'terra-enzyme-intl';
 import ShowHide from '../../src/ShowHide';
 
+jest.mock('uuid', () => ({ v4: () => '00000000-0000-0000-0000-000000000000' }));
```

Comment:
This mock needs to be reset.

Reply:
Can you get an example if it's not what's done here: https://github.com/cerner/terra-core/pull/3838/commits/aa41c4b79d15cae9e48fece9b1bcbdc462dbe454

Reply:
I'm okay with doing this in a separate PR as suggested here:
https://github.com/cerner/terra-core/pull/3838/files#r1291684878